book/managers_guide.md

```diff
@@ -0,0 +1,119 @@
+* Committed, skilled resource for the duration of the project and for ongoing maintenance for the life span of the product. 
+* Motivation for the project and personal development from those working on the project. 
+* Having appropriate tools available for the development and deployment of the project. 
+* A plan for requirement collection and transitioning the product into business as usual. 
+
+It is common for this kind of transformation to identify quality issues in the current process. This should be seen as an opportunity to improve the quality of the process.
+
+[The Government Service Standard](https://www.gov.uk/service-manual/service-standard) outlines best practices for creating public services, which includes analysis. You should use this when designing and managing the development of analysis.
+
+The rest of this page describes the benefits of doing analysis as code and aims to help you ensure that your team are applying the good quality assurance practices outlined in the wider [Quality assurance of code for analysis and research guidance](https://best-practice-and-impact.github.io/qa-of-code-guidance/intro.html). Code that applies these good practices is referred to as a reproducible analytical pipeline (RAP).
+
+Analysis team managers should not need an in-depth understanding of the code produced by their team. However, they should be able to asses whether adequate quality assurance is being applied to the development of the analysis.
+
+## Analysis as code is beneficial
+
+Carrying out analysis as code has many benefits:
+* Reproducible, quality analysis.
+* Greater efficiency and timeliness, through automation.
+* Transparent analysis and quality assurance, increasing trust.
+* Improved business continuity and knowledge management.
+
+Automating a process using code is not sufficient to achieve all of the benefits, without also applying adequate quality assurance.
+
+## Analytical code requires quality assurance
+
+We recommend that you ask your team the following questions to understand the quality assurance that is being applied in their analysis. Good quality assurance should be applied throughout an analysis project, therefore, it is important that the areas outlined below are reviewed regularly.
+
+The practices outlined below should be applied proportionately to the business risk and complexity of the project. Analytical code that doesn't apply quality assurance carries similar risk to other analysis appraoches. Your team should aim to outline what adequate quality assurance looks like early on in a project. These questions can then be used throughout the project to assess whether this target is being met, or whether the team are moving towards this goal.
+
+### Why have your chosen to do things this way?
+
+* The methodology and data should be suitable for the question being asked.
+* Your team should be using open-source analysis tools, with a reason for each given tool being chosen.
```

Comment:
Your team should use open-source analysis tools. They should be able to explain why they have chosen the tools they have and why they are confident that they are appropriate and fit for purpose.